runner/runners/facade.go

```diff
@@ -0,0 +1,13 @@
+package runners
```

Comment:
Added a bunch of comments at the top of files whose purpose is non-obvious; wdyt?